app/controllers/api/v2x1.rb

```diff
@@ -0,0 +1,47 @@
+module Api
+  module V2x1
+    class RootController < ApplicationController
+      def openapi
+        render :json => ::Insights::API::Common::OpenApi::Docs.instance["2.1"].to_json
+      end
+    end
+
+    # Same behavior as api/v1.0/
```

Comment:
These should inherit from v2.0, not v1.0

Reply:
done 👍

Reply:
The comment is incorrect now.  It can be removed though.

Reply:
yes, removed, thanks 👍